### ISSUE TITLE
Ansible Galaxy does not have the latest version

### ISSUE THREAD

**Issue by justaskz** [avanov/ansible-galaxy-pyenv#63]
Hello, I noticed that Ansible Galaxy lists 1.1.0 as a latest version, while 1.2.0 is the latest on GitHub. Could you please push 1.2.0 to the Ansible Galaxy?

**Comment by avanov** [comment]
Hi!

Unfortunately i cannot do that, since some time ago Galaxy allows only logins via GitHub and they require granting access to all metadata related to Organisations attached to my primary GitHub account, which I cannot allow myself for security reasons. There's also no legitimate technical reason why Galaxy should require Organisation data access for role  uploads.

![Screenshot 2021-08-12 at 15 33 55](https://user-images.githubusercontent.com/601955/129197865-30973b44-ee43-4e75-944d-3c8d7ab62eb4.png)
![Screenshot 2021-08-12 at 15 34 10](https://user-images.githubusercontent.com/601955/129197881-841ad466-a3af-4033-83cd-32b7699cffec.png)

**Comment by justaskz** [comment]
So maybe you can use another account or something?

**Comment by avanov** [comment]
Yes, but as I'm not actively using Ansible anymore (I switched to Nix), managing a separate account on github just for it is not too appealing to me. I don't mind if you publish the role under your name, and you could automate upstream updates of your fork if you don't want to cut the ties with the main repository.

It also doesn't mean that I am abandoning the repository, it's just I don't find Galaxy with its recent changes to authentication very appealing. There are other means how Ansible roles could be delivered to your infrastructure repository, without compromising on precision and safety features.
